includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -1012,7 +1037,7 @@ private function get_stripe_supported_payment_methods() {
 	public function payment_fields() {
 		try {
 			$display_tokenization    = $this->supports( 'tokenization' ) && is_checkout() && $this->saved_cards;
-			$show_optimized_checkout = $this->oc_enabled && $this->is_valid_optimized_checkout_page();
+			$show_optimized_checkout = $this->should_use_optimized_checkout_payment_method_layout();
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $show_optimized_checkout. Keep variable name length under 20. (undefined)

(LongVariable)